sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -110,9 +110,11 @@ class Connection(object):
     DEFAULT_DATABASE = 'master'
     DEFAULT_DRIVER = 'SQL Server'
     DEFAULT_DB_KEY = 'database'
+    DEFAULT_SQLSERVER_VERSION = "2019"
```

Comment:
I think this should be something like `1e9` or some junk value. If `self.server_version` is used for other things later, it might cause some confusion if the actual server_version is 2016.

Reply:
ah yeah, good point